### PR TITLE
Dev/user login

### PR DESCRIPTION
**Pull request by kevinleung56**
* Added staff functionality
* Everything's working EXCEPT filtering POST
   * It loops for some reason
Would love some assistance on that :)

### CODE REVIEW

**Comment by seant1** [comment]
Am I supposed to be able to log in? cos it doesn't seem to be working for me

File: R1Project/src/java/uts/isd/model/Staff.java [hunk 1]
```diff
@@ -170,7 +170,7 @@ public void setManager(String manager) {
         this.manager = manager;
     }
 
-    public boolean isActive() {
+    public boolean getActive() {
```
Comment (seant1) [inline R1Project/src/java/uts/isd/model/Staff.java:173]: pretty sure this should be isActive() - boolean getter convention. you can rename/refactor all references in netbeans

**Review by seant1 — CHANGES_REQUESTED** [review]
LGTM! I definitely understood everything you have here. Just suggested a small name change

**Comment by kevinleung56** [comment]
Really? You should be able to login provided you entered a valid email or
registered.

**Comment by seant1** [comment]
Hmm... not too sure what the problem is.. I tried recreating the db but I still can't get it to work. Maybe you try remaking the db - i can't think of anything else that would be different. Or maybe I'm just doing it wrong

**Pull request merged**